README.md

```diff
@@ -87,7 +87,7 @@ additional metrics!
 >	* kube_node_status_capacity_nvidia_gpu_cards
 >	* kube_node_status_allocatable_nvidia_gpu_cards
 >
->	are in alpha stage and will be deprecated when the kubernetes accelerator feature support is deprecated in version v1.11.
+>	are in alpha stage and will be deprecated when the kubernetes accelerator feature support is completely deprecated and removed in version v1.11(Kubernetes accelerator support is already deprecated in v1.10).
```

Comment:
I would say:

```
are deprecated and will be completely removed when the Kubernetes accelerator feature support is removed in version v1.11. (Kubernetes accelerator support is already deprecated in v1.10).
```

Reply:
@brancz @mindprince As for the deprecation statement about gpu metrics, i suggest kube-state-metrics announce it like this:
```
Kubernetes accelerator support is already deprecated in Kubernetes v1.10 and will be completely removed in version v1.11. As Kubernetes has a six month(i.e., two minor release) support window, kube-state-metrics will keep these gpu metrics until Kubernetes v1.13 is released. 
```

Reply:
not sure about that, I think we should just remove them as well, and people who really need them in that way can use the older version of kube-state-metrics

Reply:
Ok. Then we should state it as [what @mindprince said above](https://github.com/kubernetes/kube-state-metrics/pull/417#discussion_r179923665)